src/main/java/nobita/task/TaskList.java

```diff
@@ -0,0 +1,136 @@
+package nobita.task;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+/**
+ * Class that encapsulates TaskList.
+ * TaskList contains all tasks in an ArrayList.
+ *
+ * @author Zheng Chenglong
+ */
+public class TaskList implements Iterable<Task> {
+
+    /** The list that contains all tasks */
+    ArrayList<Task> tasks;
+
+    /**
+     * Constructs TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Getter for number of tasks.
+     * @return An integer representing number of total tasks.
+     */
```

Comment:
You should add an **empty line** between the method description and the parameter descriptions.